conda-nighres.yml

```diff
@@ -1,36 +1,99 @@
 name: nighres
 channels:
-  - mrtrix3
-  - simpleitk
-  - itk
   - conda-forge
-  - bioconda
   - defaults
 dependencies:
-  - python=3.9
-  - pip
-  - numpy
-  - nibabel
-  - psutil
-  - jcc
-  - pandas
-  - pyyaml
-  - scikit-image
-  - scikit-learn
-  - Nilearn
-  - statsmodels
-  - matplotlib
-  - Pillow
-  - plotly
-  - charset-normalizer
-  - chart-studio
-  - idna
-  - requests
-  - retrying
-  - urllib3
-  - tenacity
-  - webcolors
-  - gcc_linux-64
-  - gxx_linux-64
+  - _libgcc_mutex=0.1=conda_forge
+  - _openmp_mutex=4.5=1_gnu
+  - binutils_impl_linux-64=2.35.1=h27ae35d_9
+  - binutils_linux-64=2.35.1=h454624a_30
+  - blas=1.0=mkl
+  - bottleneck=1.3.4=py39hce1f21e_0
+  - brotlipy=0.7.0=py39h27cfd23_1003
+  - ca-certificates=2022.3.29=h06a4308_0
+  - certifi=2021.10.8=py39h06a4308_2
+  - cffi=1.15.0=py39hd667e15_1
+  - charset-normalizer=2.0.4=pyhd3eb1b0_0
+  - cryptography=36.0.0=py39h9ce1e76_0
+  - gcc_impl_linux-64=9.3.0=h6df7d76_17
+  - gcc_linux-64=9.3.0=h1ee779e_30
+  - gxx_impl_linux-64=9.3.0=hbdd7822_17
+  - gxx_linux-64=9.3.0=h7e70986_30
+  - h5py=3.6.0=py39ha0f2276_0
+  - hdf5=1.10.6=hb1b8bf9_0
+  - idna=3.3=pyhd3eb1b0_0
+  - intel-openmp=2021.4.0=h06a4308_3561
+  - jcc=3.11=py39he80948d_2
+  - joblib=1.1.0=pyhd3eb1b0_0
+  - kernel-headers_linux-64=2.6.32=he073ed8_15
+  - ld_impl_linux-64=2.35.1=h7274673_9
+  - libffi=3.3=he6710b0_2
+  - libgcc-devel_linux-64=9.3.0=hb95220a_17
+  - libgcc-ng=11.2.0=h1d223b6_14
+  - libgfortran-ng=7.5.0=ha8ba4b0_17
+  - libgfortran4=7.5.0=ha8ba4b0_17
+  - libgomp=11.2.0=h1d223b6_14
+  - libstdcxx-devel_linux-64=9.3.0=hf0c5c8d_17
+  - libstdcxx-ng=11.2.0=he4da1e4_14
+  - mkl=2021.4.0=h06a4308_640
+  - mkl-service=2.4.0=py39h7f8727e_0
+  - mkl_fft=1.3.1=py39hd3c417c_0
+  - mkl_random=1.2.2=py39h51133e4_0
+  - ncurses=6.3=h7f8727e_2
+  - nibabel=3.2.2=pyhd8ed1ab_0
+  - nilearn=0.9.0=pyhd8ed1ab_1
+  - numexpr=2.8.1=py39h6abb31d_0
+  - openjdk=8.0.152=h7b6447c_3
+  - openssl=1.1.1n=h7f8727e_0
+  - packaging=21.3=pyhd3eb1b0_0
+  - pandas=1.4.1=py39h295c915_1
+  - pip=21.2.4=py39h06a4308_0
+  - pycparser=2.21=pyhd3eb1b0_0
+  - pydicom=2.3.0=pyh6c4a22f_0
+  - pyopenssl=22.0.0=pyhd3eb1b0_0
+  - pyparsing=3.0.4=pyhd3eb1b0_0
+  - pysocks=1.7.1=py39h06a4308_0
+  - python=3.9.11=h12debd9_2
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python_abi=3.9=2_cp39
+  - pytz=2021.3=pyhd3eb1b0_0
+  - readline=8.1.2=h7f8727e_1
+  - requests=2.27.1=pyhd3eb1b0_0
+  - scikit-learn=1.0.2=py39h51133e4_1
+  - scipy=1.7.3=py39hc147768_0
+  - setuptools=58.0.4=py39h06a4308_0
+  - six=1.16.0=pyhd3eb1b0_1
+  - sqlite=3.38.2=hc218d9a_0
+  - sysroot_linux-64=2.12=he073ed8_15
+  - threadpoolctl=2.2.0=pyh0d69192_0
+  - tk=8.6.11=h1ccaba5_0
+  - tzdata=2022a=hda174b7_0
+  - urllib3=1.26.8=pyhd3eb1b0_0
+  - wheel=0.37.1=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7f8727e_4
   - pip:
-    - antspyx
+    - antspyx==0.3.2
+    - chart-studio==1.1.0
+    - cycler==0.11.0
+    - dipy==1.5.0
```

Comment:
Added dipy as a dependency as it seems needed for some of the example.

Tempted to actually add it as dependency in `setup.py` because it is directly imported in some nighres functions.

Reply:
@piloubazin WDYT?

Reply:
yes, I forgot to add it, it's a dependency for the DOTS algorithm (diffusion tract labeling). I would have gone without it, but other people seem to like external libraries ;P

Reply:
done